2.7/Dockerfile

```diff
@@ -23,6 +23,17 @@ RUN yum install -y centos-release-scl && \
     rpm -e --nodeps centos-logos && \
     yum clean all -y
 
+# -- generated by https://github.com/rhcarvalho/envdiff, do not edit manually --
+# The ENV instruction below is equivalent to:
+#   scl enable python27
+ENV MANPATH=/opt/rh/python27/root/usr/share/man: \
+    X_SCLS=python27  \
+    LD_LIBRARY_PATH=/opt/rh/python27/root/usr/lib64 \
+    PATH=/opt/rh/python27/root/usr/bin:$PATH \
+    XDG_DATA_DIRS=/opt/rh/python27/root/usr/share \
+    PKG_CONFIG_PATH=/opt/rh/python27/root/usr/lib64/pkgconfig
```

Comment:
For RPMs security team pointed out that enable scriptlet needs to avoid exporting path with empty path segment (usually a trailing ':'). It's done by appending envar only if it is already defined, e.g. `export LD_LIBRARY_PATH=/opt/rh/ruby200/root/usr/lib64${LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}}`

IMO we should do the same in the case of docker images too.